stylesheets/stylesheet.css

```diff
@@ -616,4 +616,16 @@ footer p {
     h4.card-title {
         font-size: 0.7em;
     }
+}
+img:hover{
```

Comment:
Please don't add the rule for all `img` tags. Add a cirtain class to contributor images and add the rule to that specific class.